.github/workflows/news-post-lpb.yml

```diff
@@ -0,0 +1,219 @@
+        env:
+          LPB_HOST: "dev-api.va.gov"
+        run:
+          for n in `cat changed-files/changed-files.txt | grep 'content/website/news/articles/\|content/website/news/digital-media/'`; do
+          echo "Posting $n to LPB";
+          DATE_STRING=`echo $n | 's/^.*date:\s*//'`;
+          TITLE_STRING=`cat $n | 's/^.*title:\s*//'`;
+          TARGET_NEWS_CATEGORY=`echo $n | sed -E 's/.*\/([a-z\-]{1,})\/.*/\1/'`;
+          SOURCE_STRING=`cat $n | 's/^.*source:\s*//'`;
+          URL_STRING=`cat $n | 's/^.*url:\s*//'`;
+          echo "Target URL https://$LPB_HOST/internal/platform-backend/v0/news/categories/$TARGET_NEWS_CATEGORY/items";
+          curl
+          --request POST
+          --header 'Content-Type:application/json'
+          --header "Authorization:Bearer ${{ env.bearer_token }}"
+          --data "{\"date\":\"$DATE_STRING\",\"title\":\"$TITLE_STRING\",\"source\":\"$SOURCE_STRING\",\"url\":\"$URL_STRING\"}"
+          https://$LPB_HOST/internal/platform-backend/v0/news/categories/$TARGET_NEWS_CATEGORY/items;
+          done;
+          for n in `cat changed-files/changed-files.txt | grep content/website/news/releases`; do
+          echo "Posting $n to LPB";
+          DATE_STRING=`echo $n | 's/^.*date:\s*//'`;
+          TITLE_STRING=`cat $n | 's/^.*title:\s*//'`;
+          TARGET_NEWS_CATEGORY=`echo $n | sed -E 's/.*\/([a-z\-]{1,})\/.*/\1/'`;
+          URL_STRING=`cat $n | 's/^.*url:\s*//'`;
+          echo "Target URL https://$LPB_HOST/internal/platform-backend/v0/news/categories/$TARGET_NEWS_CATEGORY/items";
+          curl
+          --request POST
+          --header 'Content-Type:application/json'
+          --header "Authorization:Bearer ${{ env.bearer_token }}"
+          --data "{\"date\":\"$DATE_STRING\",\"title\":\"$TITLE_STRING\",\"url\":\"$URL_STRING\"}"
+          https://$LPB_HOST/internal/platform-backend/v0/news/categories/$TARGET_NEWS_CATEGORY/items;
+          done;
+      - name: Comment on PR
+        uses: actions/github-script@v6
+        with:
+          script: |
+            github.rest.issues.createComment({
+              issue_number: context.issue.number,
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              body: 'These changes have been pushed to [dev](https://dev-developer.va.gov/about/news).'
+            })
+      - name: Previously Merged Warning Comment on PR
+        if: needs.gather_content.outputs.rename_warning == 'TRUE'
+        uses: actions/github-script@v6
+        with:
+          script: |
+            github.rest.issues.createComment({
+              issue_number: context.issue.number,
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              body: 'If this release note was merged to the `main` branch on another PR, you will need to reach out to [Team Okapi](https://lighthouseva.slack.com/archives/C01931CFMTQ) to remove the copy that remains on the previous date.'
```

Comment:
This needs to be updated to reference it being news, not release notes. Also the ending of this error talks about the date. Are we doing any deduplication of news posts on the LPB side? Would this error ever hit?